### PR TITLE
docs(module:cascader): add an FAQ section

### LINKED ISSUE #5098: cascader: add an FAQ section to address some issues
This is frequently asked question about the cascader component. Adding a FAQ section would help more users who have the same problem.

https://stackoverflow.com/questions/60320913/ng-zorro-cascader-lazy-load-data-nzloaddata-function-got-this-undefined/60928983#60928983

### PR DESCRIPTION
**Pull request by hullis**
close #5098

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by simplejason** [comment]
这个提示仍然需要存在吗？需要的话直接合了。

**Comment by hullis** [comment]
> 这个提示仍然需要存在吗？需要的话直接合了。

需要的，麻烦合一发

**Review by simplejason — APPROVED** [review]
LGTM

File: components/cascader/doc/index.en-US.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+J
```
Comment (hullis) [inline components/cascader/doc/index.en-US.md:1]: Shit...

**Pull request merged**